.github/workflows/continuous-delivery.yaml

```diff
@@ -19,14 +19,8 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v3.4.1
         with:
-          node-version: 12
-      - name: Cache dependencies
-        uses: actions/cache@v3
-        with:
-          key: npm-${{ hashFiles('./package-lock.json') }}
-          path: ~/.npm
-          restore-keys: |
-            npm-
+          cache: 'npm'
```

Comment:
there's a built-in cache in setup-node now, so I updated this similar to how we have in other repos